broker/src/main/java/org/apache/rocketmq/broker/processor/SendMessageProcessor.java

```diff
@@ -577,6 +577,7 @@ private RemotingCommand handlePutMessageResult(PutMessageResult putMessageResult
         messageExtBatch.setFlag(requestHeader.getFlag());
         MessageAccessor.setProperties(messageExtBatch, MessageDecoder.string2messageProperties(requestHeader.getProperties()));
         messageExtBatch.setBody(request.getBody());
+        messageExtBatch.setPropertiesString(requestHeader.getProperties());
```

Comment:
IMO, set properties has been done in this code `MessageAccessor.setProperties(messageExtBatch, MessageDecoder.string2messageProperties(requestHeader.getProperties()));`, so we don't need add a new variable named propertiesString to MessageExtBatch to do this thing.

Reply:
Yes, I agree with you. My Intention was to avoid another transformation since there was already a transformed string.  I thought that MessageExtBrokerInner is out of the same consideration.